app/models/profile.rb

```diff
@@ -18,6 +18,9 @@ class Profile < ApplicationRecord
   validates :ref_id, uniqueness: {
     scope: %i[account_id benchmark_id external]
   }, presence: true
+  validates :ref_id, uniqueness: {
+    scope: %i[account_id benchmark_id policy_id]
```

Comment:
@akofink Wouldn't this uniqueness create a problem when there is an existing `external: true` profile and a user wants to create a new internal profile for the same benchmark? I'll try to test this.

Reply:
Confirmed, this is a breaking change.

I'm getting:
```
Response not successful: Received status code 422
#<ActiveRecord::RecordInvalid: Validation failed: Ref has already been taken>
```

Reply:
Good catch!